Packs/HackerOne/Integrations/HackerOne/HackerOne.py

```diff
@@ -0,0 +1,588 @@
+    """
+
+    client.program_list(params={"page[size]": 1})
+
+    if demisto.params().get('isFetch'):
+        fetch_incidents(client, {})
+
+    return 'ok'
+
+
+def fetch_incidents(client: Client, last_run: dict, ) -> Tuple[dict, list]:
+    """Fetches incidents from HackerOne.
+
+    :type client: ``Client``
+    :param client: client to use
+
+    :type last_run: ``Dict[str, str]``
+    :param last_run: Last run returned by function demisto.getLastRun
+
+    :type params: ``Dict[str, str]``
+    :param params: Arguments for fetch-incident.
+
+    :rtype: ``Tuple``
+    :return: tuple of dictionary of next run and list of fetched incidents
+    """
+    validate_fetch_incidents_parameters(client.max_fetch, client.program_handle, client.filters)
+    fetch_params = prepare_fetch_incidents_parameters(client.max_fetch, client.first_fetch, client.program_handle,
+                                                      client.severity, client.state, client.filters)
+    first_fetch = fetch_params["filter[created_at__gt]"]
+
+    fetch_params["page[number]"] = last_run.get("next_page", 1)
+    fetch_params["filter[created_at__gt]"] = last_run.get("current_created_at", first_fetch)  # type: ignore
+
+    response = client.report_list(params=fetch_params)
+
+    results = response.get('data', [])
+
+    if not results:
+        next_run = last_run
+        next_run["current_created_at"] = last_run.get("next_created_at", first_fetch)
+        next_run["next_page"] = 1
+        return next_run, []
+
+    next_run = {"next_page": fetch_params["page[number]"] + 1,
+                "current_created_at": fetch_params["filter[created_at__gt]"],
+                "next_created_at": results[-1].get("attributes", {}).get("created_at")}
+
+    report_ids = last_run.get("report_ids", [])
+
+    if next_run.get("next_created_at") != last_run.get("next_created_at"):
+        report_ids = []
+
+    incidents = []
+    for result in results:
+        if result.get("id") in report_ids:
+            continue
+        report_ids.append(result.get("id"))
+        incidents.append({
+            'name': result.get('attributes', {}).get('title', ''),
+            'occurred': result.get('attributes', {}).get('created_at'),
+            'rawJSON': json.dumps(result)
+        })
+
+    next_run["report_ids"] = report_ids
+
+    return next_run, incidents
+
+
+def hackerone_program_list_command(client: Client, args: Dict[str, str]) -> CommandResults:
+    """
+    Retrieves detailed information of all the programs that the user is a member of.
+
+    :type client: ``Client``
+    :param client: Client object to be used.
+
+    :type args: ``Dict[str, str]``
+    :param args: The command arguments provided by the user.
+
+    :return: Standard command result.
+    :rtype: ``CommandResults``
+    """
+
+    validate_common_args(args)
+    params: Dict[str, Any] = {"page[size]": args.get("page_size", 50), "page[number]": args.get("page_number")}
```

Comment:
We want the same values that were validated will be passed on to the params (which is not the same as reading them twice from the args dict)

```suggestion
    page_size = arg_to_number(args.get("page_size", 50))
    page_number = arg_to_number(args.get("page_number"))
    validate_common_args(page_size=page_size, page_number=page_number)
    params: Dict[str, Any] = {"page[size]": page_size, "page[number]": page_number}    
```